src/components/UserSignIn/index.tsx

```diff
@@ -1,11 +1,12 @@
 import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
-import { UserSignIn } from '../../types';
+import { UserSignIn } from '../../interfaces';
```

Comment:
molto molto overkill definire questo tipo in un file lontanissimo da qui

lo usi solo qui, e' il tipo dello state, lo puoi definire benissimo qui dentro o al massimo in un file subito accanto, non 2 cartelle sopra

Reply:
https://htmx.org/essays/locality-of-behaviour/